resolvers/utils/metricsTemplate.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable quote-props */
+
+/**
+ * Object that contains the stats we acquire
```

Comment:
This is a template that the uploadYear resolver deep copies from. If the copied object doesn't match the structure of this at the end of the resolver, then something is up.